pkg/config/config_test.go

```diff
@@ -1036,6 +1036,14 @@ func TestTxnTotalSizeLimitValid(t *testing.T) {
 }
 
 func TestConflictInstanceConfig(t *testing.T) {
+	// Save original global state
```

Comment:
can you explain why these lines are needed? I see this function does not call `UpdateGlobal`